server/config.py

```diff
@@ -71,3 +71,6 @@
 QUEUE_POP_DESIRED_PLAYERS = int(os.getenv('QUEUE_POP_DESIRED_PLAYERS', 8))
 # How many previous queue sizes to consider
 QUEUE_POP_TIME_MOVING_AVG_SIZE = int(os.getenv('QUEUE_POP_TIME_MOVING_AVG_SIZE', 5))
+
+LOGIN_THROTTLER_CAPACITY = int(os.getenv('LOGIN_THROTTLER_CAPACITY', -1))
+LOGIN_THROTTLER_RATE_PER_S = int(os.getenv('LOGIN_THROTTLER_RATE_PER_S', -1))
```

Comment:
What happens if I set `LOGIN_THROTTLER_CAPACITY` but forget to set `LOGIN_THROTTLER_RATE_PER_S`?

Maybe this should have a reasonable positive default value?

Reply:
I would use full name `LOGIN_THROTTLER_RATE_PER_SECOND`